nemo/collections/tts/losses/aligner_loss.py

```diff
@@ -58,9 +58,7 @@ def forward(self, attn_logprob, in_lens, out_lens):
         # Convert to log probabilities
         # Note: Mask out probs beyond key_len
         key_inds = torch.arange(max_key_len + 1, device=attn_logprob.device, dtype=torch.long)
-        attn_logprob.masked_fill_(
-            key_inds.view(1, 1, -1) > key_lens.view(1, -1, 1), -float("inf")  # key_inds >= key_lens+1
-        )
+        attn_logprob.masked_fill_(key_inds.view(1, 1, -1) > key_lens.view(1, -1, 1), -1e15)  # key_inds >= key_lens+1
         attn_logprob = self.log_softmax(attn_logprob)
```

Comment:
Should we make the masking logit value a parameter to the class, similar to "bank_logit"?

Reply:
Also I am curious if anyone knows why this would produce NaN. Does it happen immediately or a long time into training?

Sounds like a pretty large bug, if it happens for any fp32 training run.

Reply:
In `FastPitch and Mixer-TTS Training` tutorial, this would appear immediately. In my own FastPitch pre-training, this appear after several epochs. I am not sure how this happen.